perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue

```diff
@@ -19,14 +19,24 @@
               class="white">
               <v-flex d-flex xs12>
                 <v-card flat>
-                  <v-card-text class="subtitle-2 grey--text pa-2">{{ this.$t('exoplatform.perkstore.title.myOrders') }}</v-card-text>
+                  <v-card-text class="subtitle-2 grey--text pa-2">
+                    <span :class="firstLoadingPerkStoreOrder && 'skeleton-text skeleton-background skeleton-header skeleton-border-radius'">
+                      {{ this.$t('exoplatform.perkstore.title.myOrders') }}
```

Comment:
no usage of `this.` inside Vue template, you 'll can use `$t('exoplatform.perkstore.title.myOrders')`

Reply:
Done!